mesonbuild/dependencies/base.py

```diff
@@ -304,6 +304,8 @@ def __deepcopy__(self, memo: T.Dict[int, 'InternalDependency']) -> 'InternalDepe
         for k, v in self.__dict__.items():
             if k in {'libraries', 'whole_libraries'}:
                 setattr(result, k, copy.copy(v))
+            elif k == 'include_directories':
+                setattr(result, k, copy.copy(self.get_include_dirs()))
```

Comment:
nitpick: you do `self.get_include_dirs().copy()` below, and `copy.copy(self.get_include_dirs())` here.... I think I prefer using the `copy()` method of `list`, since it is available